### PR TITLE
Updated multi arch docker image value for trivy, syft, grype task

### PR DESCRIPTION
**Pull request by varsha-aaynure**
# Changes
Updated value for tekton.dev/platforms annotations and mutli arch docker images for trivy, syft and grype

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Follows the [authoring recommendations][authoring]
- [ ] Includes [docs][docs] (if user facing)
- [ ] Includes [tests][tests] (for new tasks or changed functionality)
  - See the [end-to-end testing documentation][e2e] for guidance and CI details.
- [x] Meets the [Tekton contributor standards][contributor] (including functionality, content, code)
- [x] Commit messages follow [commit message best practices][commit]
- [ ] Has a kind label. You can add one by adding a comment on this PR that
  contains `/kind <type>`. Valid types are bug, cleanup, design, documentation,
  feature, flake, misc, question, tep
- [ ] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [ ] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [ ] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [ ] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [ ] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [ ] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md) for more details._

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]: https://github.com/tektoncd/catalog/issues/413
[authoring]: https://github.com/tektoncd/catalog/blob/main/recommendations.md
[docs]: https://github.com/tektoncd/community/blob/master/standards.md#docs
[tests]: https://github.com/tektoncd/community/blob/master/standards.md#tests
[e2e]: https://github.com/tektoncd/catalog/blob/main/CONTRIBUTING.md#end-to-end-testing
[contributor]: https://github.com/tektoncd/community/blob/main/standards.md
[commit]: https://github.com/tektoncd/community/blob/master/standards.md#commit-messages

### CODE REVIEW

File: task/grype/0.1/grype.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ spec:
       description: The Arguments to be passed to Grype command.
       type: array
     - name: GRYPE_IMAGE
-      default: docker.io/anchore/grype@sha256:40c41e43eff0eb979375ea705f0cf5f6f698941099f63c890c3eb90b6ef72db5 # v0.47.0
+      default: docker.io/anchore/grype@sha256:ce8398ad2028c2181b4d626e1902cf98e1adde22ff6dca9530deada21b1a9b97
```
Comment (afrittoli) [inline task/grype/0.1/grype.yaml:26]: Could you leave a comment with the version like before?

File: task/syft/0.1/syft.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ spec:
       description: The Arguments to be passed to Syft command.
       type: array
     - name: SYFT_IMAGE
-      default: docker.io/anchore/syft@sha256:42ede68d8a8762e42715692221a212cba262ce3a252ed962abf3d44ce3e68a52 # v0.55.0
+      default: docker.io/anchore/syft@sha256:dd40bc09ff2bf3b860c399e7cf5c2db088e84c85937ac05fdeca3cd88fe2a728
```
Comment (afrittoli) [inline task/syft/0.1/syft.yaml:25]: Could you leave a comment with the version like before?

File: task/trivy-scanner/0.2/trivy-scanner.yaml [hunk 3]
```diff
@@ -26,7 +26,7 @@ spec:
       description: The Arguments to be passed to Trivy command.
       type: array
     - name: TRIVY_IMAGE
-      default: docker.io/aquasec/trivy@sha256:dea76d4b50c75125cada676a87ac23de2b7ba4374752c6f908253c3b839201d9
+      default: docker.io/aquasec/trivy@sha256:944a044451791617cc0ed2ee4d1942a4f66b790d527fcd0575a6b399ccbc05a1
```
Comment (afrittoli) [inline task/trivy-scanner/0.2/trivy-scanner.yaml:29]: Could you leave a comment with the version?

File: task/grype/0.1/grype.yaml [hunk 4]
```diff
@@ -23,7 +23,7 @@ spec:
       description: The Arguments to be passed to Grype command.
       type: array
     - name: GRYPE_IMAGE
-      default: docker.io/anchore/grype@sha256:40c41e43eff0eb979375ea705f0cf5f6f698941099f63c890c3eb90b6ef72db5 # v0.47.0
+      default: docker.io/anchore/grype@sha256:ce8398ad2028c2181b4d626e1902cf98e1adde22ff6dca9530deada21b1a9b97  # latest
```
Comment (vinamra28) [inline task/grype/0.1/grype.yaml:26]: generally we should avoid taking image from the latest tag as we won't get to know the version of the binary in future. Can you please take any image digest from a tagged version?

File: task/syft/0.1/syft.yaml [hunk 5]
```diff
@@ -22,7 +22,7 @@ spec:
       description: The Arguments to be passed to Syft command.
       type: array
     - name: SYFT_IMAGE
-      default: docker.io/anchore/syft@sha256:42ede68d8a8762e42715692221a212cba262ce3a252ed962abf3d44ce3e68a52 # v0.55.0
+      default: docker.io/anchore/syft@sha256:dd40bc09ff2bf3b860c399e7cf5c2db088e84c85937ac05fdeca3cd88fe2a728  # latest
```
Comment (vinamra28) [inline task/syft/0.1/syft.yaml:25]: same as above

File: task/trivy-scanner/0.2/trivy-scanner.yaml [hunk 6]
```diff
@@ -26,7 +26,7 @@ spec:
       description: The Arguments to be passed to Trivy command.
       type: array
     - name: TRIVY_IMAGE
-      default: docker.io/aquasec/trivy@sha256:dea76d4b50c75125cada676a87ac23de2b7ba4374752c6f908253c3b839201d9
+      default: docker.io/aquasec/trivy@sha256:944a044451791617cc0ed2ee4d1942a4f66b790d527fcd0575a6b399ccbc05a1  # latest
```
Comment (vinamra28) [inline task/trivy-scanner/0.2/trivy-scanner.yaml:29]: same as above

**Review by vinamra28 — COMMENTED** [review]
thanks for taking this up. Have added a comment.

**Comment by vinamra28** [comment]
also post resolving the comments, can you please squash the PRs?

**Review by afrittoli — COMMENTED** [review]
Thank you!
/lgtm

File: task/syft/0.1/syft.yaml (hunk 5 above)
Comment (varsha-aaynure) [reply task/syft/0.1/syft.yaml]: @vinamra28 i've updated PR with above suggestions, Thanks !

**Review by vinamra28 — APPROVED** [review]
thanks

**Pull request merged**